packages/@react-types/shared/src/dnd.d.ts

```diff
@@ -114,6 +114,39 @@ interface DroppableCollectionDropEvent extends DropEvent {
   target: DropTarget
 }
 
+interface DroppableCollectionInsertDropEvent {
+  items: DropItem[],
+  dropOperation: DropOperation,
+  target: {
+    key: Key,
+    dropPosition: Omit<DropPosition, 'on'>
```

Comment:
Have a feeling the doc generator might not understand this... Were these necessary or would just keeping `DropTarget` as the type for all of them work?

Reply:
Not necessary persay, I can try replacing them with RootDropTarget/ItemDropTarget